src/modules/new-request-form/fields/LookupField.tsx

```diff
@@ -0,0 +1,245 @@
+            setInputValue(selectedOption.name);
+            onChange(selectedOption.value);
+            setOptions([selectedOption]);
+          }
+        }
+        return;
+      }
+
+      if (inputValue !== undefined) {
+        setInputValue(inputValue);
+        const searchParams = new URLSearchParams();
+        searchParams.set("name", inputValue.toLocaleLowerCase());
+        searchParams.set("source", "zen:ticket");
+        searchParams.set("field_id", fieldId.toString());
+        searchParams.set("requester_id", userId.toString());
+        if (organizationId !== null)
+          searchParams.set("organization_id", organizationId);
+        setIsLoadingOptions(true);
+        try {
+          const response = await fetch(
+            `/api/v2/custom_objects/${customObjectKey}/records/autocomplete?${searchParams.toString()}`
+          );
+
+          const data = await response.json();
+          if (data !== undefined) {
+            setOptions(
+              data.custom_object_records.map(
+                ({ name, id }: { name: string; id: string }) => ({
+                  name,
+                  value: id,
+                })
+              )
+            );
+          } else {
+            setOptions([]);
+          }
+        } catch (error) {
+          return error;
+        } finally {
+          setIsLoadingOptions(false);
+        }
+      }
+      return;
+    },
+    [customObjectKey, userId, organizationId, fieldId, onChange, options]
+  );
+
+  const debounceHandleChange = useMemo(
+    () => debounce(handleChange, 300),
+    [handleChange]
+  );
+
+  useEffect(() => {
+    return () => debounceHandleChange.cancel();
+  }, [debounceHandleChange]);
+
+  useEffect(() => {
+    if (value) {
+      fetchSelectedOption(value as string);
+    }
+  }, []);
+
+  const onFocus = () => {
+    setInputValue("");
+    selectedOption && setOptions([selectedOption]);
+  };
+
+  const handleKeyDown = useCallback((event) => {
+    if (event.key === "Enter") {
+      event.preventDefault();
+    }
+  }, []);
+
+  return (
+    <GardenField>
+      <Label>
+        {label}
+        {required && <Span aria-hidden="true">*</Span>}
+      </Label>
+      {description && (
+        <Hint dangerouslySetInnerHTML={{ __html: description }} />
+      )}
+      <input type="hidden" name={name} value={selectedOption?.value} />
+      <Combobox
+        inputProps={{ required }}
+        startIcon={<SearchIcon />}
+        validation={error ? "error" : undefined}
+        inputValue={inputValue}
+        selectionValue={selectedOption?.value}
+        isAutocomplete
+        placeholder={t(
+          "new-request-form.lookup-field.placeholder",
+          `Search ${label.toLowerCase()}`
+        )}
+        onFocus={onFocus}
+        onChange={debounceHandleChange}
+        renderValue={() =>
+          selectedOption ? selectedOption?.name : EMPTY_OPTION.name
+        }
+        style={{ marginTop: "8px" }}
```

Comment:
We shouldn't override Garden styles. Especially with a static value of `8px`. The margin-top is missing just because we have the hidden input between the `Label` and the `Combobox`. Let's move the hidden input after the combobox

Reply:
Ok, I moved hidden input after the combobox and removed the style.